docs/mkdocs.yml

```diff
@@ -795,12 +807,18 @@ extra_css:
     - 'css/styles2.css'
 markdown_extensions:
     - admonition
+    - meta
     - toc:
           permalink: true
     - pymdownx.highlight
     - pymdownx.snippets
     - pymdownx.superfences
+    - pymdownx.tabbed:
+        alternate_style: true 
     - pymdownx.tabbed
+    - pymdownx.emoji:
+          emoji_index: !!python/name:materialx.emoji.twemoji
+          emoji_generator: !!python/name:materialx.emoji.to_svg
```

Comment:
👀........😍😍😍😍

Reply:
true easteregg... specially for you @Adam-it  😂